Description: Adjust path to rc?.d directories
 On SUSE, they are at /etc/init.d/rc?.d/, but on Debian they are at
 /etc/rc?.d/.
Forwarded: not-needed

Index: chkconfig-10.3.90/chkconfig
===================================================================
--- chkconfig-10.3.90.orig/chkconfig	2008-04-10 10:58:47.000000000 +0200
+++ chkconfig-10.3.90/chkconfig	2008-04-10 10:58:55.000000000 +0200
@@ -136,7 +136,7 @@
 sub initlinks_rc {
   my $l;
   for $l (keys %to_d) {
-    my @links = grep {s/^S\d\d//} ls("$initdir/$to_d{$l}");
+    my @links = grep {s/^S\d\d//} ls("$initdir/../$to_d{$l}");
     $links{$l} = { map {$_ => 1} @links };
   }
   %links_unknown = ();
